safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -160,55 +164,73 @@ def __init__(self, ethereum_client: EthereumClient, redis: Redis):
             maxsize=4096, ttl=60 * 30
         )  # 2 hours of caching
 
+    def get_redis_metadata_timeout_key(self, uri: str):
+        return f"metadata_timeout:{uri}"
+
     @cachedmethod(cache=operator.attrgetter("cache_uri_metadata"))
     @cache_memoize(
-        60 * 60 * 24,
+        60 * 60 * 24 * 2,
```

Comment:
Increase the metadata time because is immutable, we should decide which would be an appropriate value.

Reply:
Sure, it's inmutable, but why going from 1 day to 2 days?

Reply:
Imeant that we could increase the time that we storage the metadata to for examples 2 days, what do you think?

Reply:
No strong opinion there, but I think 2 days should be alright too